app/templates/services/_base_service_page.html

```diff
@@ -59,9 +59,6 @@
       {% import "toolkit/summary-table.html" as summary %}
       {% for section in sections %}
         {{ summary.heading(section.name, id=section.slug) }}
-        {% if section.editable %}
```

Comment:
Removing this section-level Edit link breaks all the old frameworks... (e.g. you can't edit any DOS2 services any more if you happen to have DOS2 open) it *might* be OK as they are closed for applications now anyway, but I'd be tempted to leave it in.  The trick then would be to get the `/create/` route to work OK with `edit_section: false`, i.e. get it to render as `/edit/service-name/service-name` rather than just `/edit/service-name` as it does at the moment.